financial-connections/src/main/java/com/stripe/android/financialconnections/FinancialConnectionsSheetViewModel.kt

```diff
@@ -102,6 +108,36 @@ internal class FinancialConnectionsSheetViewModel @Inject constructor(
         }
     }
 
+    private fun nativeAuthFlowKillSwitchActive(synchronizeSessionResponse: SynchronizeSessionResponse): Boolean {
+        return synchronizeSessionResponse.manifest.features?.any { it.key == "bank_connections_mobile_native_version_killswitch" && it.value }
+            ?: true
```

Comment:
Also we can convert these to receiver functions to make it a bit nicer :) 
```suggestion
    private fun SynchronizeSessionResponse.nativeAuthFlowKillSwitchActive(): Boolean {
        return manifest.features?.any { it.key == "bank_connections_mobile_native_version_killswitch" && it.value }
            ?: true
```

Reply:
Since I moved to a domain object, it makes sense to keep it as a param I think?